bigquery_etl/deploy.py

```diff
@@ -119,5 +119,10 @@ def _create_or_update(
         )
         log.info(f"{table} updated.")
     else:
-        client.create_table(table)
+        try:
+            client.create_table(table)
+        except NotFound as e:  # Raised when project/dataset doesn't exist.
+            raise FailedDeployException(
+                f"Unable to create {table} in missing dataset/project"
+            ) from e
```

Comment:
Usually I'd disagree because of how broad `Exception` is, but since it's a one-line `try` block and we're chaining the exceptions anyway I think it should be fine 👍 

A Python wiki entry on this: https://wiki.python.org/moin/HandlingExceptions

Reply:
The current test should still be relevant after fbf312a28b9d13c7c9feea9dccb0d973aed267df as a special case example